### PR TITLE
Fixed the overlay issue in styling

### LINKED ISSUE #361: [Bug] Graph variables and units overlay issue
**Describe the bug**
Accessing the graph page. Add a dataset. The settings block has a weird overlay of the X and Y labels with the pull-down menu indicator (triangle).

Text should be legible.
Screenshot from 2021-02-02 21-32-20

### PR DESCRIPTION
**Pull request by laithAwad**
I have used overriding the muiselect padding to fix the overlay issue

### CODE REVIEW

File: Client/src/appTheme.tsx [hunk 1]
```diff
@@ -51,5 +51,11 @@ export const classStyles = () => {
     divider: {
       background: theme.palette.primary.light,
     },
+    muiSelectArrow: {
```
Comment (GD-Dheer) [inline Client/src/appTheme.tsx]: Good fix.

**Review by GD-Dheer — APPROVED** [review]

File: Client/src/Components/Graph/GraphControls/Axes/AxisStateControl.tsx [hunk 2]
```diff
@@ -105,27 +106,30 @@ export const AxisStateControl = (props: IProps) => {
...
+        <FormControl fullWidth>
           <InputLabel id={`${axisName}VariableLabel`}>{axisName} Variable</InputLabel>
           <Select
             labelId={`${axisName}Variable`}
+            fullWidth
             id={`${axisName}Variable`}
             value={axisState.variableName}
             autoWidth={true}
             onChange={handleVariableChange}
+            label="Asynchronous"
```
Comment (ds161833) [inline Client/src/Components/Graph/GraphControls/Axes/AxisStateControl.tsx]: i think the label has no effect here does it? what does it do

File: Client/src/Components/Graph/GraphControls/Axes/AxisStateControl.tsx [hunk 3]
```diff
@@ -5,6 +5,7 @@ import React, { useState } from "react"
 
 import { IAxisStateModel } from '../../../../Models/Graph/IGraphStateModel'
 import SnackbarUtils from "../../../Utils/SnackbarUtils"
+import { classStyles } from '../../../../appTheme'
```
Comment (ds161833) [inline Client/src/Components/Graph/GraphControls/Axes/AxisStateControl.tsx]: unused import

**Comment by laithAwad** [comment]
Screen Shot 2021-03-15 at 11 04 15 PM
Screen Shot 2021-03-15 at 11 04 23 PM

File: Client/src/Components/Graph/GraphControls/Axes/AxisStateControl.tsx (hunk 2 above)
Comment (laithAwad) [reply Client/src/Components/Graph/GraphControls/Axes/AxisStateControl.tsx]: it was for the previous solution, removed it now

File: Client/src/Components/Graph/GraphControls/Axes/AxisStateControl.tsx (hunk 3 above)
Comment (laithAwad) [reply Client/src/Components/Graph/GraphControls/Axes/AxisStateControl.tsx]: removing it

**Review by ds161833 — APPROVED** [review]
clean

**Pull request merged**